scripts/ci/jobs/diff-dumps.sh

```diff
@@ -122,14 +122,10 @@ upload_offline_dump() {
 
     file_to_check="scanner-v4-defs-${latest_version}.zip"
 
-    if curl --silent --show-error --max-time 60 --retry 3 -o $file_to_check https://definitions.stackrox.io/v4/offline-bundles/$file_to_check; then
-        # If the file exists, add it to scanner-vuln-updates.zip
-        zip scanner-vuln-updates.zip "$file_to_check"
-        echo "$file_to_check added to scanner-vuln-updates.zip"
-    else
-        echo "$file_to_check does not exist."
-        exit 1
-    fi
+    curl --silent --show-error --fail --max-time 60 --retry 3 -o $file_to_check https://definitions.stackrox.io/v4/offline-bundles/$file_to_check
+    zip scanner-vuln-updates.zip "$file_to_check"
```

Comment:
```suggestion
    unzip -l "$file_to_check"
    zip scanner-vuln-updates.zip "$file_to_check"
```

Do you think it is worth it?